src/Microsoft.AspNetCore.Server.Kestrel.Core/KestrelServerLimits.cs

```diff
@@ -258,8 +258,8 @@ public TimeSpan RequestHeadersTimeout
         /// This can be overridden per-request via <see cref="IHttpMinRequestBodyDataRateFeature"/>.
         /// </summary>
         /// <remarks>
-        /// Defaults to 1 byte/second with a 5 second grace period.
```

Comment:
Can we add a regular non-doc comment indicating that this default comes from IIS like we do for a lot of our other limits?